### PR TITLE
[new] add lumx@0.3.96

### PR DESCRIPTION
**Pull request by Amomo**
Hi @Piicksarn 

For #6444, I add lumx at v0.3.96 and add git auto-update config in its package.json.
repo url: https://github.com/lumapps/lumX

I don't use a npm auto-update because the library need to be built after npm install.
GitHub repo do provide the `dist/` folder, so I choose to use git auto-update.

Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `127`, Star `1,431`, Fork `195`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

File: ajax/libs/lumx/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lumx",
+  "filename": "lumx.min.js",
+  "description": "The first responsive front-end framework based on AngularJS & Google Material Design specifications",
+  "kewords": [
```
Comment (PeterDaveHello) [inline ajax/libs/lumx/package.json]: keywords ...

Comment (Amomo) [inline ajax/libs/lumx/package.json]: oops! I have corrected the typo. thanks :-D

**Comment by Amomo** [comment]
@Piicksarn would you mind checking this PR for me? Thank you~ :-D

**Comment by PeterDaveHello** [comment]
test failed ...

**Comment by Piicksarn** [comment]
It's fine!
@PeterDaveHello, would you please have a look at the pr?
Thanks!

**Comment by PeterDaveHello** [comment]
no fonts?

**Comment by Amomo** [comment]
oh, yes, we need fonts.
@Piicksarn would you mind checking this PR for me? Thank you~ :-D

**Comment by Piicksarn** [comment]
@Amomo, the pr is find but rebase to master is required, would you please rebase it and ping me again?
thanks!

File: ajax/libs/lumx/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/lumapps/lumx.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lumapps/lumx.git",
+    "basePath": "dist",
+    "files": [
+      "ilumx*.+(js|css)",
```
Comment (PeterDaveHello) [inline ajax/libs/lumx/package.json]: Why not just `"ilumx*"`?

Comment (Amomo) [inline ajax/libs/lumx/package.json]: Oh! `"ilumx"` is a typo, should be `"lumx"` ...sorry!
I have corrected it and use a simple representation now.

**Comment by Amomo** [comment]
@Piicksarn I have corrected the typo and rebased my PR on latest master branch.
Could you help me checking it again? Thank you! :-)

File: ajax/libs/lumx/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/lumapps/lumx.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lumapps/lumx.git",
+    "basePath": "dist",
+    "files": [
+      "lumx*",
```
Comment (PeterDaveHello) [inline ajax/libs/lumx/package.json]: This can not match files before v0.3.x, use "**/lumx*" will be better.

Comment (PeterDaveHello) [inline ajax/libs/lumx/package.json]: See 97a04a4

**Pull request merged**